Description: Makes sources compile/compatible with bouncycastle 1.49, original is compatible with 1.50
Index: libjitsi/src/org/jitsi/impl/neomedia/transform/dtls/TlsServerImpl.java
===================================================================
--- libjitsi.orig/src/org/jitsi/impl/neomedia/transform/dtls/TlsServerImpl.java	2014-06-03 05:17:55.252148000 -0400
+++ libjitsi/src/org/jitsi/impl/neomedia/transform/dtls/TlsServerImpl.java	2014-06-03 05:18:17.544148000 -0400
@@ -29,7 +29,7 @@
     private final CertificateRequest certificateRequest
         = new CertificateRequest(
                 new short[] { ClientCertificateType.rsa_sign },
-                /* supportedSignatureAlgorithms */ null,
+                /* supportedSignatureAlgorithms  null, */
                 /* certificateAuthorities */ null);
 
     /**
